velox/common/base/BloomFilter.h

```diff
@@ -31,9 +34,27 @@ namespace facebook::velox {
 // expected entry, we get ~2% false positives. 'hashInput' determines
 // if the value added or checked needs to be hashed. If this is false,
 // we assume that the input is already a 64 bit hash number.
-template <bool hashInput = true>
+// case:
+// InputType can be one of folly hasher support type when hashInput = false.
+// InputType can only be uint64_t when hashInput = true.
+template <class InputType = uint64_t, bool hashInput = true>
```

Comment:
`InputType` should be added at methods `insert` and `mayContain` since the class itself does not need it.

Reply:
fixed